lib/granite/version.rb

```diff
@@ -1,3 +1,3 @@
 module Granite
-  VERSION = '0.8.2'.freeze
```

Comment:
Don't BUMP version. we'll do it on our own whenile preparing a release

Reply:
This bug produce inconsistent models in our main project, that leads to rollbar errors.
What is the fastest way to get this fix into our main app?

Reply:
Master contains some changes that are planned for 0.9.0 so we'll create separate branch 0.8.x where we'll backport all the fixes like that. For now just remove version bump and we'll merge it to master. Then I'll backport it to 0.8.x branch and release new minor version.

Reply:
Ping me when it happens cause we need to fix all broken opportunities (merge https://github.com/toptal/platform/pull/25951) but this does not make sense until we fix the source of inconsistency.